mlflow/store/db_migrations/versions/de4033877273_add_evaluation_datasets.py

```diff
@@ -0,0 +1,125 @@
+"""add evaluation datasets
```

Comment:
The migration script for the evaluation datasets is slightly different: https://github.com/mlflow/mlflow/blob/genai-dataset/mlflow/store/db_migrations/versions/de4033877273_add_evaluation_datasets.py 

eval dataset tags table (to support search functionality without incurring index-readability hits to that table since we're not storing as JSONB in PG) is the main difference.

Reply:
100% agreed. I'll merge master to the feature branch then fix the migration lineage in the feature branch directly and propagate the changes.